### ISSUE TITLE
Survey task: Choice component content is skipped in a screen reader 

### ISSUE THREAD

**Issue by eatyourgreens** [zooniverse/front-end-monorepo#2455]
Discovered while working on #2449. The focus tests, like this one, highlight an accessibility issue:
```
with choice without more than 1 image, confusions, or questions
      ✓ should have the "Identify" button as the document active element
```

We have at least one volunteer who uses NVDA, and contributes to survey task projects. If focus jumps straight to the Identify button, then the rest of the Choice component content won't be read out, and they'll miss it. That makes the survey task inaccessible to blind volunteers. It could be a regression too, since the survey task is accessible in PFE.

_Originally posted by @eatyourgreens in https://github.com/zooniverse/front-end-monorepo/issues/2449#issuecomment-923831229_

**Comment by eatyourgreens** [comment]
Marcy Sutton posted a study of [accessible client-side in-page navigation](https://marcysutton.com/prototype-testing-accessible-clientside-routing/), which might also be useful here.
